cpp/include/cuspatial/types.hpp

```diff
@@ -26,6 +26,13 @@
 
 namespace cuspatial {
 
+/**
+ * @addtogroup type_definition_operator
+ * @brief Type declaration for cuspatial
+ * @{
+ * @file
+ */
+
 /**
  * @brief A timestamp
```

Comment:
I don't think this struct is used anymore. We should consider removing it and this file.

Reply:
Would you like to do that in this PR?

Reply:
If we're going to remove this anyway I wouldn't bother adding docs. The current group name is uninformative will only confuse users, especially if they'll never need to know about it.